handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -118,6 +122,13 @@
             }
         }
 
+        if (cause == null) {
+            if (!isNettyTcnative()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1066)